Interface/SamuraiHudWindow.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Numerics;
+using Dalamud.Game.ClientState.Structs.JobGauge;
+using Dalamud.Plugin;
+using ImGuiNET;
+
+namespace DelvUIPlugin.Interface {
+    public class SamuraiHudWindow : HudWindow {
+        public override uint JobId => 38;
+        
+        private new static int BarHeight => 26;
+        private new static int BarWidth => 357;
+        private new static int XOffset => 178;
+        private new static int YOffset => 496;
+        
+        public SamuraiHudWindow(DalamudPluginInterface pluginInterface, PluginConfiguration pluginConfiguration) : base(pluginInterface, pluginConfiguration) { }
+
+        protected override void Draw(bool _) {
+            DrawHealthBar();
+            DrawPrimaryResourceBar();
+            DrawSenResourceBar();
+            DrawMeditationResourceBar();
+            DrawTargetBar();
+        }
+
+        protected override void DrawPrimaryResourceBar()
+        {
+            var gauge = PluginInterface.ClientState.JobGauges.Get<SAMGauge>();
+
+            // const int xPadding = 5;
```

Comment:
Let's try to keep commented out code out of the repo;  everything is in source control so we can always bring it back if we need it.